### PR TITLE
Fix general metrics namespace and labels

### PR DESCRIPTION
**Pull request by leviska**
Fix actix metrics, so they will have the same name across all services with constant label pointing to specific service. It helps making one generic dashboard with parameter

### CODE REVIEW

File: libs/blockscout-service-launcher/src/metrics.rs [hunk 1]
```diff
@@ -11,12 +11,13 @@ pub struct Metrics {
 impl Metrics {
     pub fn new(service_name: &str, endpoint: &str) -> Self {
         let registry = prometheus::default_registry();
-        let metrics_middleware =
-            PrometheusMetricsBuilder::new(&(service_name.to_owned() + "_metrics"))
-                .registry(registry.clone())
-                .endpoint(endpoint)
-                .build()
-                .unwrap();
+        let const_labels = HashMap::from([("rust_service".into(), service_name.into())]);
```
Comment (rimrakhimov) [inline libs/blockscout-service-launcher/src/metrics.rs:14]: Nit: maybe just:

```suggestion
        let const_labels = HashMap::from([("service".into(), service_name.into())]);
```
?

**Review by rimrakhimov — APPROVED** [review]

**Review by sevenzing — APPROVED** [review]

File: libs/blockscout-service-launcher/src/metrics.rs [hunk 2]
```diff
@@ -11,12 +11,13 @@ pub struct Metrics {
...
+        let metrics_middleware = PrometheusMetricsBuilder::new("rust_microservices")
```
Comment (rimrakhimov) [inline libs/blockscout-service-launcher/src/metrics.rs:15]: Nit:
```suggestion
        let metrics_middleware = PrometheusMetricsBuilder::new("rust_microservices_metrics")
```
?

**Review by rimrakhimov — APPROVED** [review]

File: libs/blockscout-service-launcher/src/metrics.rs (hunk 1 above)
Comment (leviska) [reply libs/blockscout-service-launcher/src/metrics.rs:14]: Prometheus will automatically add "service" label equal to kubernetes label, so it's better to have different name

File: libs/blockscout-service-launcher/src/metrics.rs (hunk 2 above)
Comment (leviska) [reply libs/blockscout-service-launcher/src/metrics.rs:15]: It's already a metric, don't see a reason why we need to add it
What else could be with this prefix? rust_microservices_???

File: libs/blockscout-service-launcher/src/metrics.rs (hunk 1 above)
Comment (rimrakhimov) [reply libs/blockscout-service-launcher/src/metrics.rs:14]: Get it. Maybe `service_name` in that case? I just don't like that we use `rust_` prefix in both the metric's name and the label

File: libs/blockscout-service-launcher/src/metrics.rs (hunk 2 above)
Comment (rimrakhimov) [reply libs/blockscout-service-launcher/src/metrics.rs:15]: Make sense! Just thought about that from the historic perspective, as we used `metrics` suffix before

**Pull request merged**